core_types/Cargo.toml

```diff
@@ -21,6 +21,7 @@ futures-util-preview = "=0.3.0-alpha.12"
 
 reed-solomon = "0.2.1"
 rust-base58 = "0.0.4"
+rustc-serialize = "0.3.24"
```

Comment:
What is this needed for? It really shouldn't be used. This crate has been deprecated in favor of `serde` about 2 years ago.

Reply:
Thanks for catching that. I made the changes to update to `serde` :)